package.json

```diff
@@ -179,13 +182,16 @@
     "csso": "^4.0.3",
     "electron": "^8.2.5",
     "electron-builder": "^22.6.0",
+    "electron-devtools-installer": "^3.0.0",
```

Comment:
Just so I understand this: This is a package that enables us to install any of the supported additional DevTools extensions, right? Does this download ALL available devtools or only the ones we explicitly specify? Don't know about the space impact though

Reply:
Yes, it's only a loader and doesn't come with the tools themselves. Pretty lightweight at 16kb.

Reply:
Perfect!